### DISCUSSION TITLE
source code of cosmograph.app/run

### DISCUSSION THREAD

**Discussion by revayand** [cosmograph-org/cosmograph-issues#6] [General]
Hi
I want to develop and app using cosmos lib that is something like cosmograph.app/run. Can I access the source code of "cosmograph.app/run" ? . Is there any open source SPA ?

I have seen the codesandbox example at https://github.com/cosmograph-org/cosmos/wiki. But it is very simple.

Thanks

**Comment by rokotyan** [comment]
Hi @revayand, the code of the Cosmograph app is not open sourced. But we're planning to release of its components soon as a library. Can you tell more what project you are working on and what component you wanted to use?

**Reply by revayand** [reply]
I want to develop an app with a layering feature that can load multiple metadata files. I also want to remove the pre-processing of the Cosmograph app to make it faster. and such cases

**Reply by rokotyan** [reply]
@revayand Can you send us a note to hi@cosmograph.app, so we could give you access to the beta version of our new library?
